### PR TITLE
Web 38/forgot password

### PR DESCRIPTION
**Pull request by arsalan96**
Worked on fixes in login screen (added/removed unnecessary styles)
added missing prop in button component
added capitalize style in text field generic component
worked on forgot screen UI

### CODE REVIEW

File: src/pages/forgot.tsx [hunk 1]
```diff
@@ -1,27 +1,71 @@
...
+		bottom: 0,
+		right: 0,
+		left: 0,
+		backgroundImage: `url(${loginBackground.src})`,
+		backgroundRepeat: 'no-repeat',
+		backgroundSize: 'cover',
+		backgroundPosition: 'center'
+	},
+
+	resetButton: {
```
Comment (UmairZahid490) [inline src/pages/forgot.tsx:37]: Remove unused style classes

File: src/components/common/Button.tsx [hunk 2]
```diff
@@ -29,7 +30,12 @@ export const GenericButton: FunctionComponent<ButtonProps> = ({
 	...props
 }) => {
 	return (
-		<Button sx={[classes.loginButton, { ...style }]} onClick={onClick} {...props}>
+		<Button
+			sx={[classes.loginButton, { ...style }]}
+			onClick={onClick}
+			disabled={disabled}
```
Comment (UmairZahid490) [inline src/components/common/Button.tsx:36]: Don't need to add disabled prop {...prop} will handle it.

**Review by UmairZahid490 — CHANGES_REQUESTED** [review]

File: src/pages/forgot.tsx (hunk 1 above)
Comment (arsalan96) [reply src/pages/forgot.tsx:37]: Fixed

File: src/components/common/Button.tsx (hunk 2 above)
Comment (arsalan96) [reply src/components/common/Button.tsx]: Fixed

**Review by UmairZahid490 — APPROVED** [review]

**Pull request merged**